lib/src/firestore.dart

```diff
@@ -3,6 +3,7 @@
 
 import 'dart:async';
 import 'dart:js';
+import 'dart:typed_data';
```

Comment:
Should we also add `export 'dart:typed_data' show Uint8List;` so that users are not forced to import this dependency all the time?

We could also make `Blob` expect a regular `List<int>` in the constructor and internally convert it to `Uint8List` so that we hide this detail completely. E.g.:

```dart
class Blob {
  final Uint8List _data;
  Blob(List<int> data) : _data = new Uint8List.fromList(data);
  List<int> get data => _data;
}
```

Not sure which is better though. WDYT?

Reply:
I like that and then we don't have to export dart:typed_data